addons/mrp/tests/test_bom.py

```diff
@@ -2443,6 +2444,54 @@ def test_archive_operation(self):
         self.assertEqual(bom.bom_line_ids[1].operation_id, ope_2)
         self.assertEqual(bom.byproduct_ids[1].operation_id, ope_2)
 
+    def add_variant_to_product(self, product):
+        attr = self.env['product.attribute'].create({'name': 'Attribute'})
+        self.env['product.attribute.value'].create({
+            'name': 'Variant', 'attribute_id': attr.id, 'sequence': 1
```

Comment:
imo It's a bit weird to create an attribute for one value only
It makes the attribute a bit useless